package.json

```diff
@@ -18,6 +18,7 @@
 	"categories": [
 		"Themes"
 	],
+	"icon": "images/icon.png",
```

Comment:
Please add yourself to the contributors list here as well 🙏

Reply:
Added!